CHANGELOG-2.1.md

```diff
@@ -1,4 +1,14 @@
-# Changelog since v2.0.0
+# v2.1.1 (Changelog since v2.1.0)
+
+## Bug Fixes
```

Comment:
my apologies. I took a second thought and think it might be better to have the bugs listed in the ##Bug Fixes section, WDYT?
lgtm otherwise

Reply:
"Snapshot Controller" is a subsection under "Bug Fixes" to indicate they are bugs fixed in snapshot controller.  List bugs separately under "Bug Fixes" and above "Snapshot Controller" does not look right to me if that is what you meant.  I'd like to follow the convention of other items in the changelog.